content/docs/api-reference/make_checkpoint.md

```diff
@@ -86,8 +86,8 @@ Experiment results have been applied to your workspace.
 
 In this example we kill the process (with Ctrl + C) after 3 checkpoints (at 0,
 100, and 200). The <abbr>cache</abbr> will contain those 3 versions of
-`int.txt`. And DVC applies the last checkpoint to the <abbr>workspace</abbr>
-even when more cycles happened before the interrupt:
+`int.txt`. DVC applies the last checkpoint to the <abbr>workspace</abbr> even
+when more cycles happened before the interrupt:
```

Comment:
```suggestion
`int.txt`. DVC applies the last checkpoint to the <abbr>workspace</abbr>
regardless of how many cycles happened before the interrupt:
```
I don't understand the current sentence, so I'm not sure if this captures the intended meaning.

Reply:
Changing to "if more cycles happened after that" but this example is kind of a placeholder for now. I'll prob steal/adapt your GS examples for the other docs in a following iteration 😬 Thanks though!